alpine-chroot-install

```diff
@@ -115,15 +115,14 @@ ewarn() {
 
 # Writes Alpine APK keys embedded in this script into directory $1.
 dump_alpine_keys() {
-	local dest_dir="$1"
-	local content id line
+	dest_dir="$1"
```

Comment:
`local` is perfectly valid. It’s not specified by POSIX Shell Command Language, but all relevant shell implementations support it. Removing it would make the script less robust.

Reply:
shellcheck using shebang to determine capability of each shell, and it does not just randomly give warnings as you are thinking.

but anyway, i'm going to revert shellcheck changes, since it's out of scope was supposed to be a good to have for someone who is interested in shellcheck.

Reply:
> shellcheck using shebang to determine capability of each shell, and it does not just randomly give warnings as you are thinking.

shellcheck cannot know which shell implementations (dialect) the script author decided to support. `/bin/sh` is the most generic, typically a symlink for the actual shell.

See [shellcheck(1)](https://www.mankier.com/1/shellcheck#-s).

Reply:
So?
/bin/sh is guaranteed to be _at least_ bourne shell compatible. linter works with the strict definition of shebang as it should.